packages/twenty-server/src/modules/webhook/jobs/call-webhook-jobs.job.ts

```diff
@@ -41,51 +27,64 @@ export class CallWebhookJobsJob {
   ) {}
 
   @Process(CallWebhookJobsJob.name)
-  async handle(data: CallWebhookJobsJobData): Promise<void> {
+  async handle(
+    data: WorkspaceEventBatch<ObjectRecordBaseEvent>,
+  ): Promise<void> {
     // If you change that function, double check it does not break Zapier
     // trigger in packages/twenty-zapier/src/triggers/trigger_record.ts
+    // Also change the openApi schema for webhooks
+    // packages/twenty-server/src/engine/core-modules/open-api/utils/computeWebhooks.utils.ts
 
     const webhookRepository =
       await this.twentyORMGlobalManager.getRepositoryForWorkspace<WebhookWorkspaceEntity>(
         data.workspaceId,
         'webhook',
       );
 
-    const nameSingular = data.objectMetadataItem.nameSingular;
-    const operation = data.operation;
-    const eventName = `${nameSingular}.${operation}`;
+    const [nameSingular, operation] = data.name.split('.');
 
     const webhooks = await webhookRepository.find({
       where: [
-        { operations: ArrayContains([eventName]) },
+        { operations: ArrayContains([data.name]) },
```

Comment:
not clear what data.name is. Looks to be eventName
eventName = objectName.operation

Reply:
data seems to actually be eventBatch: { eventName, events, workspaceId } I would prefer this naming